docs/usage/instance_set.md

```diff
@@ -0,0 +1,109 @@
+
+Now Check NCM instances in the cluster
+```bash
+$ kubectl get instance
+NAME                      KIND
+test-instance-set-b57zs   Instance.v1alpha1.node.sapcloud.io
+test-instance-set-c4bg8   Instance.v1alpha1.node.sapcloud.io
+test-instance-set-kvskg   Instance.v1alpha1.node.sapcloud.io
+```
+Now you will see N (number of replicas specified in the manifest) new instances whose name are prefixed with the instance-set object name that you created.
+
+After a few minutes (~3 minutes for AWS), you would see that new nodes have joined the cluster. You can see this using
+```bash
+$ kubectl get nodes
+NAME                                         STATUS    AGE       VERSION
+ip-10-250-0-234.eu-west-1.compute.internal   Ready     3m        v1.8.0
+ip-10-250-15-98.eu-west-1.compute.internal   Ready     3m        v1.8.0
+ip-10-250-6-21.eu-west-1.compute.internal    Ready     2m        v1.8.0
+``` 
+This shows how new nodes have joined your cluster
+
+## Inspect status of instance-set
+
+To inspect the status of any created instance, you can always do a 
+```bash
+$ kubectl get instanceset test-instance-set -o yaml
+apiVersion: node.sapcloud.io/v1alpha1
+kind: InstanceSet
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"node.sapcloud.io/v1alpha1","kind":"InstanceSet","metadata":{"annotations":{},"name":"test-instance-set","namespace":"","test-label":"test-label"},"spec":{"minReadySeconds":200,"replicas":3,"selector":{"matchLabels":{"test-label":"test-label"}},"template":{"metadata":{"labels":{"test-label":"test-label"}},"spec":{"class":{"kind":"AWSInstanceClass","name":"test-aws"}}}}}
+  clusterName: ""
+  creationTimestamp: 2017-12-27T08:37:42Z
+  finalizers:
+  - node.sapcloud.io/operator
+  generation: 0
+  initializers: null
+  name: test-instance-set
+  namespace: ""
+  resourceVersion: "12630893"
+  selfLink: /apis/node.sapcloud.io/v1alpha1/test-instance-set
+  uid: 3469faaa-eae1-11e7-a6c0-828f843e4186
+spec:
+  instanceClass: {}
+  minReadySeconds: 200
+  replicas: 3
+  selector:
+    matchLabels:
+      test-label: test-label
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        test-label: test-label
+    spec:
+      class:
+        kind: AWSInstanceClass
+        name: test-aws
+status:
+  availableReplicas: 3
+  fullyLabeledReplicas: 3
+  instanceSetCondition: null
+  lastOperation:
+    lastUpdateTime: null
+  observedGeneration: 0
+  readyReplicas: 3
+  replicas: 3
+```
+
+## Health monitoring
+
+- If you try to delete/terminate any of the instances backing the instance-set, by either talking to the NCM or from the cloud provider. The NCM recreates a matching healthy instance to replace the deleted instance. 
+- Similarily, if any of your instances are unreachable or in an unhealthy state (kubelet not ready / disk pressure) for over a timeout period (~ 5mins), the NCM recreates health nodes to replace them.
```

Comment:
for longer than the configured timeout ...

... the NCM recreates the nodes to replace the unhealthy nodes.